src/cpp/core/Database.cpp

```diff
@@ -945,7 +945,7 @@ bool SchemaUpdater::parseVersionOfFile(const FilePath& file, SchemaVersion* pVer
    boost::split(split, fileStem, boost::is_any_of("_"));
    if (split.size() != 3)
    {
-      LOG_WARNING_MESSAGE("Unrecognized file in the db folder, it will be skipped: " + file.getAbsolutePath());
+      LOG_DEBUG_MESSAGE("Unrecognized file in /usr/lib/rstudio-server/db, it will be skipped: " + file.getAbsolutePath());
```

Comment:
Just a nit, but I think we'll already know the directory name from the absolute path right?  Or I suppose we could log the migrationsPath_ and use file.getFilename()?  It might be helpful to call out that this is an older schema file. How about something like this:

```suggestion
     if (split.size() == 2)
         LOG_DEBUG_MESSAGE("Not applying sql schema file from previous release: " + file.getAbsolutePath());
     else
         LOG_DEBUG_MESSAGE("Not applying unrecognized sql schema file: " + file.getAbsolutePath());
```

Reply:
Great Idea!